### PR TITLE
[Metricbeat] change server_status_path default setting for nginx module

### PR DESCRIPTION
**Pull request by goku321**
Changed default `server_status_path` to `nginx_status`

Fixes: elastic/beats#13806

### CODE REVIEW

**Comment by kaiyan-sheng** [comment]
jenkins, test this please

**Comment by exekias** [comment]
Thank you for contributing @goku321, could you please add a changelog entry? You will also need to run `mage update`

**Comment by goku321** [comment]
Sure @exekias , will do :+1:

**Comment by kaiyan-sheng** [comment]
Hi @goku321 thanks for contributing. This PR will also need a rebase/merge from master in order to let CI pass.

**Comment by goku321** [comment]
Hi @kaiyan-sheng I have synced this PR with master. Also, will add a changelog entry. Thanks for your patience

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -61,6 +61,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - system/raid metricset now uses /sys/block instead of /proc/mdstat for data. {pull}11613[11613]
 - kubernetes.container.cpu.limit.cores and kubernetes.container.cpu.requests.cores are now floats. {issue}11975[11975]
 - Add statistic option into cloudwatch metricset. If there is no statistic method specified, default is to collect Average, Sum, Maximum, Minimum and SampleCount. {issue}12370[12370] {pull}12840[12840]
+- Change `server_status_path` default setting for nginx module {issue}13806[13806], {pull}14099[14099]
```
Comment (kaiyan-sheng) [inline CHANGELOG.next.asciidoc]:
```suggestion
- Change `server_status_path` default setting for nginx module {issue}13806[13806] {pull}14099[14099]
```

Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: I wouldn't consider this as a breaking change. Maybe move this changelog entry under bugs session?

Comment (goku321) [reply CHANGELOG.next.asciidoc]: Sure 👍

Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: Thanks!

**Comment by kaiyan-sheng** [comment]
jenkins, test this please

**Comment by goku321** [comment]
I am struggling with `make update` (some python issue). Will fix it and update the pr

**Comment by kaiyan-sheng** [comment]
> I am struggling with `make update` (some python issue). Will fix it and update the pr

Maybe try `mage update` and/or `mage fmt update` under metricbeat folder 😬

**Comment by goku321** [comment]
Thanks for the suggestion @kaiyan-sheng but still getting the same error (`This backport is for Python 2.7 only.`) Trying to debug it 😅

**Comment by kaiyan-sheng** [comment]
Yeah unfortunately python2.7 is needed...

**Comment by kaiyan-sheng** [comment]
Maybe this will help if you don't want to spend too much time on fixing python2.7:
```
kaiyansheng@KaiyanMacBookPro:~/go/src/github.com/elastic/beats/metricbeat (GH-13806)$ git diff
diff --git a/metricbeat/modules.d/nginx.yml.disabled b/metricbeat/modules.d/nginx.yml.disabled
index 786cc90ed..d9833114e 100644
--- a/metricbeat/modules.d/nginx.yml.disabled
+++ b/metricbeat/modules.d/nginx.yml.disabled
@@ -9,8 +9,8 @@
   # Nginx hosts
   hosts: ["http://127.0.0.1"]
 
-  # Path to server status. Default nginx_status
-  #server_status_path: "nginx_status"
+  # Path to server status. Default server-status
+  #server_status_path: "server-status"
 
   #username: "user"
   #password: "secret"
```

**Comment by kaiyan-sheng** [comment]
Hi @goku321, could you rebase this PR by any chance and also I posted the change you need for fixing the breaking python ci test.

**Comment by goku321** [comment]
@kaiyan-sheng sure, will do it by tomorrow

**Pull request merged**